ideascube/monitoring/forms.py

```diff
@@ -110,24 +113,28 @@ class LoanForm(forms.ModelForm):
 
     def clean_specimen(self):
         barcode = self.cleaned_data['specimen']
-        if Loan.objects.due().filter(specimen__barcode=barcode).exists():
+        if (Loan.objects.due().filter(specimen__barcode=barcode).exists()
+         or Loan.objects.due().filter(specimen__serial=barcode).exists()):
```

Comment:
> WARNING: A long digression not really related to this PR

Oh, absolutely.

And until we collectively agree on a specific style and run the linter in the CI, I won't block a pull request on this.

However, it's already late, I'm curious, and your pull request was a perfect occasion to ask how you prefer aliging long `if`s. 😃 

> I would add one or two space between the `if` and the `(` to keep `Loan...` aligned and keep an indentation between `if` and `and`

My problem with this is that adding a new condition (e.g an `and` clause to the conditional from the code we're are talking under) means modifying the existing lines:

```diff
-        if (Loan.objects.due().filter(specimen__barcode=barcode).exists()
-         or Loan.objects.due().filter(specimen__serial=barcode).exists()):
+        if  (Loan.objects.due().filter(specimen__barcode=barcode).exists()
+         and ...
+          or Loan.objects.due().filter(specimen__serial=barcode).exists()):
```

That's a bigger diff than necessary, making the review that much harder.

> I'm not convinced neither by the pep8 solution

Your example is only one of the solutions suggested in PEP8. However, you're right that's it's extremely unreadable (the conditions being aligned with the inner block), and in fact the `pep8` tool dislikes that one too.

Another solution suggested in PEP8 (and which the `pep8` tool likes) is:

```python
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```

That's the solution I favour.